### PR TITLE
Milestone-1: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Debas-31**
## Portfolio: setup and mobile version skeleton.
In this project of portfolio milestone-1 :
- Added navigation section.
-  Added linters setup.
-  Added header section.
-  Added Github workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+- [GitHub](https://github.com/Debas-31)
+- [Twitter](https://twitter.com/DEBSH76956492)
+- [LinkedIn](https://www.linkedin.com/in/debas-gebrengus-5256a2159/)
+
+### 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```
Comment (DanteAlonsoHT) [inline README.md:33]: You should add the right link to go to the issue page in your repository.

- [ ] Fix link for issues page.

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+### Show your support
+
+Give a ⭐️ if you like this project!
+
+### Acknowledgments
+- Microverse
+
+## 📝 License
+
+This project is [MIT]() licensed.
```
Comment (DanteAlonsoHT) [inline README.md:44]: Also, you should add the right link to see the License for this repository.

- [ ] Fix link for License.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <a  href="#" class="down-nav-logo">S-D-Bus</a>
+            <a href="#"><i class="fas fa-bars fa-2x"></i></a>
+        </div>
+    </nav>
+    <header class="header-container">
+        <div class="header-content">
+            <h1 class="header-text"> Debas G-ngus</h1>
+            <h2>software Developer</h2>
+            <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,don't hesitate to contact me.</p>
+        </div>
```
Comment (DanteAlonsoHT) [inline index.html:37]: The container of the `section` is a section tag, please add these lines of code inside `<section></section>`

- [ ] Add your content (not header) into a `section` HTML tag.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Debas-31,

Good job so far, you did an amazing advance!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- [ ]  [OPTIONAL] There are some details to do to make your front-end design almost exactly that the Figma design you selected. If you want to be perfect, it would be amazing.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

<hr>

I saw your progress in this project, and I'm sure you have the capabilities enough to finish these changes required in time 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,44 @@
...
+![screenshot](./image/mobile-screen-shoot.png)
+
+### Built With
+
+- HTML5
+- CSS3
+
+### Live Demo
+
+[Visit the Live Demo]( https://debas-31.github.io/mobile-version-portfolo/)
```
Comment (DanteAlonsoHT) [inline README.md:19]: - [ ]  [OPTIONAL] There is a broken image in your website deployed using Github pages.

image

File: README.md [hunk 5]
```diff
@@ -1 +1,44 @@
...
+
+
+![screenshot](./image/mobile-screen-shoot.png)
+
+### Built With
+
+- HTML5
+- CSS3
+
+### Live Demo
```
Comment (DanteAlonsoHT) [inline README.md:17]: I strongly recommend you add instructions to set up this project in any local environment, check out the example below:

```
### Install

To get a local copy up and running follow these simple example steps.
- Open terminal
- Clone this project by the command `git clone URL/repository_name.git`
- `cd repository_name` folder
- Open `index.html` in your local browser or using Live Server in Visual Studio Code.
```

- [ ] [OPTIONAL] Add instructions about how to set up your project using Git, you should add these instructions for all of your next projects.

File: README.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+### Show your support
+
+Give a ⭐️ if you like this project!
+
+### Acknowledgments
+- Microverse
+
+## 📝 License
+
+This project is MIT licensed.
```
Comment (DanteAlonsoHT) [inline README.md:44]: Please, add the link to see what the MIT License means.
For instance: 
```
This project is [MIT](https://opensource.org/licenses/MIT) licensed.
```

- [ ] Add the right link for MIT License.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        <section >
+          <div class="header-content">
+            <img class="header-img" src="./image/big_ellipse.svg" alt="big-eclipse">
+            <h1 class="header-text"> Debas G-ngus</h1>
+            <h2>software Developer</h2>
+            <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,don't hesitate to contact me.</p>
+          </div>
+        </section>
+        <hr>
+    </header>
```
Comment (DanteAlonsoHT) [inline index.html:42]: `header` HTML tag should not be where your section is, you should use the `header` tag to cover the `nav` section.

- [ ] Remove `header` tag and move it to cover `nav` tag.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+</main>
```
Comment (DanteAlonsoHT) [inline index.html:43]: - [ ] Use `main` tag to only cover all your `section` tags you have.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @Debas-31, 

Great work on making the changes requested by me 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You are near to reach the completion of this project! 💯  

Cheers and Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 8 above)
Comment (Debas-31) [reply index.html:43]: Could you elaborate on this issue please?

File: README.md (hunk 2 above)
Comment (Emmyn5600) [reply README.md]: Well solved

File: README.md [hunk 9]
```diff
@@ -1 +1,44 @@
...
+![screenshot](./image/mobile-screen-shoot.png)
+
+### Built With
+
+- HTML5
+- CSS3
+
+### Live Demo
+
+[Visit the Live Demo]()
```
[the remark this answers is not in the document]
Comment (Emmyn5600) [reply README.md]: Well solved

File: README.md (hunk 4 above)
Comment (Emmyn5600) [reply README.md]: Well solved

File: README.md (hunk 5 above)
Comment (Emmyn5600) [reply README.md:17]: Well solved

File: README.md (hunk 6 above)
Comment (Emmyn5600) [reply README.md]: Well solved

File: index.html (hunk 3 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 7 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 8 above)
Comment (Emmyn5600) [reply index.html]: Well solved

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @Debas-31  

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewers.
   
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**